vtr_flow/parse/parse_config/common/abc.txt

```diff
@@ -1,10 +1,10 @@
 #ABC Metrics
-abc_depth;abc0.out;lev\s*=\s*(\d+)
+abc_depth;abc0.out;.* lev\s*=\s*(\d+)
 
 #ABC Run-time Metrics
 abc_synth_time;abc0.out;elapse: .* seconds, total: (.*) seconds
 abc_cec_time;abc.cec.out;elapse: .* seconds, total: (.*) seconds
-abc_sec_time;abc.sec.out;elapse: .* seconds, total: (.*) seconds
+abc_sec_time;abc.lec.out;elapse: .* seconds, total: (.*) seconds
```

Comment:
What does this change do?

Reply:
This changes which file it looks for this parameter. I believe by the request of Vaughn the file name was changed from abc.sec.out to abc.lec.out.

Reply:
Nope, I didn't ask for this (don't recall anything about it).  Is this the logical equivalence check runtime (and hence maybe lec is a better name than sec, which would seem to imply sequential equivalence check)?

Reply:
Oh sorry, it was Kevin, https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/1429#discussion_r456826480